evennia/utils/containers.py

```diff
@@ -202,14 +202,17 @@ def load_data(self):
         """
         if self.typeclass_storage is None:
             self.typeclass_storage = {}
-            for key, data in self.loaded_data.items():
+            for key, data in list(self.loaded_data.items()):
```

Comment:
to "snapshot" the contents of the `dict` because below I may remove `key`.

Reply:
Please don't resolve your own conversations. Presumably they are for the reviewer to see.